api/sql/schema/750$transfer.report.settlement.sql

```diff
@@ -73,16 +73,18 @@ UNION ALL SELECT
 FROM
     [transfer].vTransfer v
 JOIN
-    card.card c ON c.cardId = v.cardId
+    [card].[card] c ON c.cardId = v.cardId
+JOIN
```

Comment:
Is there any chance to have a [card].vCard instead? This will free us from doing such joins everywhere across other modules. Other modules should know minimum about how card tables are related to each other.